recipe/meta.yaml

```diff
@@ -32,4 +32,4 @@ about:
 
 extra:
   recipe-maintainers:
-    -janschulz
\ No newline at end of file
+    - janschulz
```

Comment:
In text comments: yes. In the github API: no (at least when used as orgs, like in `conda smithy register-github`)